components/form-token-field/index.js

```diff
@@ -503,6 +558,11 @@ FormTokenField.defaultProps = {
 	isBorderless: false,
 	disabled: false,
 	tokenizeOnSpace: false,
+	messages: {
+		added: __( 'Item added.' ),
```

Comment:
Given that these are assigned at load-time, I anticipate the strings are not correctly localized because `setLocaleData` for the editor is called as an inline script before `wp-edit-post` (but after `wp-components`).